frontend/src/pages/AccountManagement/index.js

```diff
@@ -0,0 +1,380 @@
+import React, { useContext, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import {
+  Alert,
+  Button,
+  Dropdown,
+  Fieldset,
+  Form,
+  Grid,
+  GridContainer,
+  Link,
+  Radio,
+} from '@trussworks/react-uswds';
+import { useForm, FormProvider, useFormContext } from 'react-hook-form';
+import { useParams } from 'react-router';
+import Avatar from '../../components/Avatar';
+import './index.scss';
+import UserContext from '../../UserContext';
+import {
+  subscribe,
+  unsubscribe,
+  updateSettings,
+  getEmailSettings,
+} from '../../fetchers/settings';
+import { requestVerificationEmail } from '../../fetchers/users';
+import EmailVerifier from './EmailVerifier';
+
+const emailPreferenceErrorMessage = 'Please select a frequency preference';
+
+// these keys should match the keys in /src/constants.js:USER_SETTINGS.EMAIL.VALUES
+const frequencyValues = [
+  { key: 'never', label: 'Do not notify me' },
+  { key: 'immediately', label: 'Immediately' },
+  { key: 'today', label: 'Daily digest' },
+  { key: 'this week', label: 'Weekly digest' },
+  { key: 'this month', label: 'Monthly digest' },
+];
+
```

Comment:
should we consider moving these kind of things to a private npm package or something so we don't have to maintain lists like this in two places?

Reply:
[Created a jira for this](https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1020)